assets/js/app.ts

```diff
@@ -147,41 +147,58 @@ const Hooks = {
     },
   },
   VideoPlayer: {
-    mounted() {
+    async mounted() {
       const backdrop = document.querySelector("#video-backdrop");
-
       this.playerId = this.el.id;
 
       // TODO: remove this once we have a better way to handle autoplay
       const autoplay = this.el.id.startsWith("analytics-") ? false : "any";
 
-      this.player = videojs(this.el, {
-        autoplay: autoplay,
-        liveui: true,
-        html5: {
-          vhs: {
-            llhls: true,
-          },
-        },
+      this.player = await window.VidstackPlayer.create({
+        target: this.el,
+        autoplay,
+        viewType: "video",
+        streamType: "on-demand",
+        logLevel: "warn",
+        crossOrigin: true,
+        playsInline: true,
+        live: true,
+        seeking: true,
+        poster:
+          "https://console.algora.io/asset/storage/v1/object/public/images/algora-gradient-90px.png",
       });
+      const getVideoProvider = (type: string) => {
+        switch (type) {
+          case "youtube":
+          case "youtube/video":
+            return (this.player.type = "youtube/video");
+
+          case "hls":
+            return (this.player.type = "application/x-mpegURL");
+
+          case "dash":
+            return (this.player.type = "application/dash+xml");
+
+          default:
+            return (this.player.type = "video/mp4");
+        }
+      };
 
       const playVideo = (opts: {
         player_id: string;
         id: string;
         url: string;
+        format: string;
         title: string;
+        type: string;
```

Comment:
Instead of format & type, let's just add an `is_live` boolean

We also need to actually provide this param in `lib/algora_web/live/player_component.ex` when we push a `play_video` event